doc/tutorial.rst

```diff
@@ -131,6 +131,16 @@ At this stage the graph ``G`` consists of 8 nodes and 3 edges, as can be seen by
     >>> G.number_of_edges()
     3
 
+.. note:: With Python 3.6 or later the behavior of Graph/DiGraph is ordered. Since dictionaries became ordered with that version, the order of adjacency reporting: G.adj, G.successors, G.predecessors is now the order of edge adding. However, the order of G.edges is not always the order of edge adding. The order of G.edges is the order of the adjacencies which includes both the order of the nodes and each node's adjacencies. See example below:
```

Comment:
I am not sure it is worth mentioning that this happened with Python 3.6, since we only work with 3.6 or later.  It is should also mention that the order is insertion order.  Maybe just:
```
The order of adjacency reporting (e.g., G.adj, G.successors, G.predecessors) is the order of edge addition. However, the order of G.edges is the order of the adjacencies which includes both the order of the nodes and each node's adjacencies. See example below:
```

Reply:
Thanks a lot for the review. I've reflected the changes.